perfkitbenchmarker/linux_benchmarks/cloudsuite_inmemory_analytics_benchmark.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs the in-memory analytics benchmark of Cloudsuite.
+
+More info: http://cloudsuite.ch/inmemoryanalytics/
+"""
+
+import re
+
+from perfkitbenchmarker import configs
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import sample
+
+flags.DEFINE_string('ima_dataset',
+                    '/data/ml-latest-small',
+                    'Dataset to use for training.')
+flags.DEFINE_string('ima_ratings_file',
+                    '/data/myratings.csv',
+                    'Ratings file to give the recommendation for.')
+FLAGS = flags.FLAGS
+
+BENCHMARK_NAME = 'cloudsuite_inmemory_analytics'
+BENCHMARK_CONFIG = """
+cloudsuite_inmemory_analytics:
+  description: >
+    Run Cloudsuite in-memory analytics benchmark. Specify the number of worker
+    VMs with --num_vms.
+  vm_groups:
+    master:
+      vm_spec: *default_single_core
+    workers:
+      vm_spec: *default_single_core
+      vm_count: 1
```

Comment:
Makes sense, just confirming that's what you wanted since it looks a bit weird. The master group, which should always have 1 VM, has the vm_count omitted (which functionally is the same as explicitly setting it to 1), while the worker group, which can have one or more VMs, has the vm_count explicitly set to 1 (but can be overridden elsewhere). Functionally, what you have here is fine, but it *looks* like the master group size is nebulous while the workers group size is recommended/fixed.